examples/gallery/plot/datetime-inputs.py

```diff
@@ -0,0 +1,58 @@
+"""
+Datetime inputs
+---------------
+
+The :meth:`pygmt.Figure.plot` method can plot datetime inputs of individual types:
```

Comment:
It's a global feature, so almost all functions (plot, text et al.) can accept datetime inputs.

Reply:
ok, so what's the best directory to place it?

Reply:
It's OK to keep the example in this directory. What I meant is this sentence "The :meth:`pygmt.Figure.plot` method can plot datetime inputs of individual types:" gives me the impression that only `plot()` can accept datetime inputs.

Reply:
> 
> 
> It's OK to keep the example in this directory. What I meant is this sentence "The :meth:`pygmt.Figure.plot` method can plot datetime inputs of individual types:" gives me the impression that only `plot()` can accept datetime inputs.

Slightly modified the text passage to make it more general.